src/components/form.jsx

```diff
@@ -0,0 +1,40 @@
+import { useDispatch } from 'react-redux';
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/book/book';
+
+const Form = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+  const addNewBook = (e) => {
+    e.preventDefault();
+    const newBook = {
+      item_id: uuidv4(),
+      title,
+      author,
+      category: 'Action',
+    };
+    dispatch(addBook(newBook));
+    setAuthor('');
+    setTitle('');
+  };
+
+  const titleInput = (e) => {
+    setTitle(e.target.value);
+  };
+
+  const authorInput = (e) => {
+    setAuthor(e.target.value);
+  };
+
+  return (
+    <form onSubmit={addNewBook}>
+      <input type="text" placeholder="Enter Title" onChange={titleInput} value={title} />
+      <input type="text" placeholder="Enter Author" onChange={authorInput} value={author} />
+      <button type="submit">Add Book</button>
+    </form>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you make all your inputs required to prevent the user to submit an empty form. You can add the `required` attribute in all your inputs. Right now, once clicked the **Add Book** button without filling in the inputs, it submits an empty form.